lib/screen/main/main_item.dart

```diff
@@ -0,0 +1,93 @@
+import 'package:flutter/material.dart';
+import 'package:showcase_flutter_build_web_desktop/extension/context_extension.dart';
+import 'package:showcase_flutter_build_web_desktop/gen/colors.gen.dart';
+
+import '../../theme/dimen.dart';
+
+class MainItem extends StatelessWidget {
+  const MainItem({super.key});
+
+  // TODO integrate with real data
+  @override
+  Widget build(BuildContext context) {
+    return Container(
+      decoration: const BoxDecoration(
+        borderRadius: BorderRadius.all(Radius.circular(Dimens.borderRadius10)),
+        color: Colors.white,
+      ),
+      child: Padding(
+        padding: const EdgeInsets.all(Dimens.space10),
+        child: Column(
+          children: [
```

Comment:
Use `crossAxisAlignment: CrossAxisAlignment.stretch` we can get rid of using `sizebox` 🙏 
```suggestion
        child: Column(
          crossAxisAlignment: CrossAxisAlignment.stretch,
          children: [
```